app/controllers/dashboard/form/publish_controller.rb

```diff
@@ -34,8 +34,10 @@ def update
           @resource.indexing_source = Proc.new { nil }
           @resource.save
           @resource.publish
-        elsif request_curation? && !@resource.draft_curation_requested
-          @resource.update_column(:draft_curation_requested, true)
+        elsif curator_action_requested?
+          column = request_curation? ? :draft_curation_requested : :accessibility_remediation_requested
+
+          @resource.update_column(column, true)
```

Comment:
I think these changes are fine.  If we want this to be a bit cleaner or easier to read, we could pull all of this curation logic out into it's own service class.

Reply:
Or i guess it will already be up in AirTable, so the CurationSyncService will exclude it.

Reply:
Ah nevermind. That's only for published works.

Reply:
My understanding is that curation will include accessibility review (it's currently listed as one of the services in the curation request description), but I agree we need to do something to make that more clear. Might be as simple as updating the description with language emphasizing that curation includes remediation so if you want both, select curation.

I'll play around with moving it to a service class. I tried just pulling it into a private method, but there were some issues with error raising/number of redirects, but I might be able to resolve those if it's a separate service class.

Reply:
Ah, I see that now in the description.  Yeah, I think it would be good to emphasize the accessibility part in the curation description a bit more.